blocks/form/form.js

```diff
@@ -0,0 +1,135 @@
+function constructPayload(form) {
+  const payload = {};
+  [...form.elements].forEach((fe) => {
+    if (fe.id) {
+      payload[fe.id] = fe.value;
+    }
+  });
+  return payload;
+}
+
+/**
+ * Once the payload is constructed and the form entry is validated,
+ * we submit the form. It creates an entry in the spreadsheet to store the form
+ * data.
+ */
```

Comment:
Following Milo patterns, we try to avoid comments and comment blocks to save space if we can.

Reply:
Done.